src/Components/Recommendation/Recommendation.js

```diff
@@ -331,26 +331,44 @@ const Recommendation = ({ rule, ack, clusters, match }) => {
                       !ackIsFetching &&
                       ackData && (
                         <React.Fragment>
-                          {intl.formatMessage(
-                            messages.ruleIsDisabledJustification
-                          )}
-                          <i>
-                            {ackData?.justification ||
-                              intl.formatMessage(messages.none)}
-                          </i>
-                          {(ackData?.updated_at || ackData?.created_at) && (
-                            <span>
-                              &nbsp;
-                              <DateFormat
-                                date={
-                                  new Date(
-                                    ackData?.updated_at || ackData?.created_at
-                                  )
+                          {ackData?.justification
+                            ? intl.formatMessage(
+                                messages.ruleIsDisabledWithJustificaiton,
+                                {
+                                  date: (
+                                    <span>
+                                      <DateFormat
+                                        date={
+                                          new Date(
+                                            ackData?.updated_at ||
+                                              ackData?.created_at
+                                          )
+                                        }
+                                        type="onlyDate"
+                                      />
+                                    </span>
+                                  ),
+                                  reason: ackData?.justification,
```

Comment:
Nit-pick: `?` is not required (you already check the presence of `justification` in the upper ternary operator).